docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.timeout_precommit_delta
+How much the timeout_precommit increases with each round.
+```toml
+timeout_precommit_delta = "500ms"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0ms"`     |
+
+### consensus.timeout_commit
+How long we wait after committing a block, before starting on the new height.
+This gives us a chance to receive some more precommits, even though we already have +2/3.
+```toml
+timeout_commit = "1s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.double_sign_check_height
+How many blocks to look back to check existence of the node's consensus votes before joining consensus.
+```toml
+double_sign_check_height = 0
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+When non-zero, the node will panic upon restart if the same consensus key was used to sign {double_sign_check_height}
+last blocks. So, validators should stop the state machine, wait for some blocks, and then restart the state machine to
+avoid panic.
+
+### consensus.skip_timeout_commit
+Make progress as soon as we have all the precommits.
+```toml
+skip_timeout_commit = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+This is similar to setting [`consensus.timeout_commit`](#consensustimeout_commit) to `"0s"`.
+
+### consensus.create_empty_blocks
+If there are no transactions in the mempool, empty blocks are proposed to indicate that the chain is still running.
+```toml
+create_empty_blocks = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+This is more relevant to networks with a low number of transactions.
+
+### consensus.create_empty_blocks_interval
+If there are no transactions in the mempool, empty blocks are proposed to indicate that the chain is still running at
+this interval.
+```toml
+create_empty_blocks_interval = "0s"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0s"`      |
+
+### consensus.peer_gossip_sleep_duration
+Consensus reactor internal sleep duration when waiting for the next piece of calculation.
+```toml
+peer_gossip_sleep_duration = "100ms"
+```
+
+| Value type          | string (duration) |
+|:--------------------|:------------------|
+| **Possible values** | &gt;= `"0ms"`     |
+
+This is a generic sleep duration for the consensus reactor that allows other threads to run when the reactor has no work
+to do.
+
+### consensus.peer_gossip_intraloop_sleep_duration
+Consensus reactor upper bound for a random sleep duration.
```

Comment:
Is this experimental?

Reply:
doesn't seem so, it is used in the consensus reactor